lib/movements.js

```diff
@@ -26,13 +26,15 @@ class Movements {
     this.digCost = 1
     this.placeCost = 1
     this.liquidCost = 1
+    this.entityCost = 1
 
     this.dontCreateFlow = true
+    this.dontMineUnderFallingBlock = true
     this.allow1by1towers = true
     this.allowFreeMotion = false
     this.allowParkour = true
     this.allowSprinting = true
-    this.dontMineUnderFallingBlock = true
```

Comment:
Is the dontMineUnderFallingBlock option removed complete now? If so this would be a breaking change. Can you back port this option for the new system?

Reply:
It is still there, I just moved it up to the top so that it is next to `this.dontCreateFlow`